geonode/settings.py

```diff
@@ -2144,3 +2144,11 @@ def get_geonode_catalogue_service():
     "document": os.getenv('DOCUMENT_EXTRA_METADATA_SCHEMA', DEFAULT_EXTRA_METADATA_SCHEMA),
     "geoapp": os.getenv('GEOAPP_EXTRA_METADATA_SCHEMA', DEFAULT_EXTRA_METADATA_SCHEMA)
 }, **CUSTOM_METADATA_SCHEMA}
+
+
+'''
+Define the URLs patterns used by the SizeRestrictedFileUploadHandler
+to evaluate if the file is greater than the limit size defined
+'''
+
+SIZE_RESTRICTED_FILE_UPLOAD_ELEGIBLE_URL_NAMES = ("data_upload", "uploads-upload", "document_upload",)
```

Comment:
I see... we should probably start splitting "internal settings" from public ones, i.e. the ones that are expected to be changed by admins rather than internal modules.

A mechanism to register new routes would be better here, but I understand it's not straightforward to implement.